FunctorCategories/examples/notebooks/FunctorCategories.ipynb

```diff
@@ -1869,8 +1869,8 @@
       "A row module over Q of rank 2\n",
       "\n",
       "Matrix: \n",
-      "[ [   5,   2,  12 ],\n",
-      "  [  10,   5,  35 ] ]\n",
+      "[ [   5/3,   5/6,  35/6 ],\n",
+      "  [  10/3,   7/6,  37/6 ] ]\n",
```

Comment:
Did those numbers change after compilation? If yes, this is obviously not a good sign.

Reply:
Yes.

Reply:
The probably some logic template does not respect the matrix equality (like in https://github.com/homalg-project/CAP_project/pull/1194#discussion_r1042180284) -> you should investigate, because the result might not be consistent.